src/main/java/csv/WritePersonCsvDemo.java

```diff
@@ -0,0 +1,17 @@
+package main.java.csv;
+
+import main.java.serialization.Person;
+import main.java.serialization.PersonService;
+
+import java.util.List;
+
+public class WritePersonCsvDemo {
+
+    private final static String PEOPLE_DATA = "people.csv";
```

Comment:
Reorder the modifiers to comply with the Java Language Specification.

Correct order: `static final`